pages/linux/larasail.md

```diff
@@ -0,0 +1,24 @@
+# larasail
+
+> A CLI tool for managing Laravel on Digital Ocean servers.
+> Homepage: <https://github.com/thedevdojo/larasail>.
+
+- Set up the server with Laravel dependencies using the default PHP version:
+
+`larasail setup`
+
+- Set up the server with Laravel dependencies using a specific PHP version:
+
+`larasail setup php71`
```

Comment:
Should php71 be in tokens since it can vary ?

Reply:
Currently it's the only one available, but yeah. 👍